pkgs/development/tools/analysis/pdepend/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, makeWrapper, lib, php }:
+let
+  pname = "pdepend";
+  version = "2.10.3";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/pdepend/pdepend/releases/download/${version}/pdepend.phar";
+    sha256 = "I5+n8+a3rHRgW4OM6FbVcf3T1wu9zBrTSb5sGq7ArH4=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    install -D $src $out/libexec/pdepend/pdepend.phar
+    makeWrapper ${php}/bin/php $out/bin/pdepend \
+      --add-flags "$out/libexec/pdepend/pdepend.phar"
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "
+      PHP Depend is an adaptation of the established Java
```

Comment:
This is a bit long for description, https://nixos.org/manual/nixpkgs/stable/#var-meta-description

Reply:
Added a `longdescription` and updated `description`.